### PR TITLE
Master -> Main Rename

### PR DESCRIPTION
**Pull request by dannylamb**
Part of https://github.com/Islandora/documentation/issues/1595

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ You can access the IIIF interface at: http://localhost:8080/cantaloupe/iiif/2/
 ### JWT
 
 Islandora 8 uses JWT for authentication across the stack. Crayfish microservices, Fedora, and Drupal all use them. 
-Crayfish and Fedora have been set up to use a master token of `islandora` to make testing easier. To use it, just set
+Crayfish and Fedora have been set up to use a main token of `islandora` to make testing easier. To use it, just set
```
Comment (rosiel) [inline README.md:101]: were these tokens changed? I'm guessing not, as this is a deep-within-the-stack use and not a github-branch-related one?

Comment (whikloj) [reply README.md:101]: I think the only thing to change is the term "master" to "main" in the above sentence. The token itself is still "islandora" and behaves the same way.

Comment (rosiel) [reply README.md:101]: in this context, 'main' doesn't seem to convey the same meaning. Would 'universal' or 'global' or 'generic' be applicable?

Comment (manez) [reply README.md:101]: Does "default" make sense in this case?

Comment (whikloj) [reply README.md:101]: I'm good with either.

Comment (rosiel) [reply README.md:101]: Default sounds good.

Comment (dannylamb) [reply README.md:101]: yeah wasn't sure about that when doing this.  i'm good with default :+1:

**Comment by dannylamb** [comment]
Should be good to go now @manez @rosiel @whikloj

**Review by rosiel — APPROVED** [review]

**Pull request merged**